src/EVEMon/MainWindow.cs

```diff
@@ -402,6 +402,14 @@ private void EveMonClient_MonitoredCharacterCollectionChanged(object sender, Eve
             UpdateTabs();
         }
 
+        /// <summary>
+        /// See MSDN.
+        /// </summary>
+        /// <param name="hWndLock"></param>
+        /// <returns></returns>
+        [DllImport("user32.dll")]
+        public static extern bool LockWindowUpdate(IntPtr hWndLock);
```

Comment:
I think this belongs in here https://github.com/peterhaneve/evemon/blob/master/src/EVEMon.Common/Controls/NativeMethods.cs

And the call to it could be placed inside the extension methods to suspend and resume drawing
https://github.com/peterhaneve/evemon/blob/master/src/EVEMon.Common/Controls/ControlExtensions.cs#L30

Reply:
Hi

You're probably right. EveMon is a bit complex to find the correct spot. The change however works nicely. The LockWindowUpdate() calls just replace the SETREDRAW based code.

FYI On non tablet mode machines you can hold alt-tab during start and see the changes this code makes.

Reply:
I absolutely agree that the change works nicely. I've been annoyed at it as well, so it's a great fix.
I haven't had the time to check out the other changes yet, but I'll post some feedback when I do.